<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PantallaMenu</class>
 <widget class="QWidget" name="PantallaMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>1920</width>
       <height>1080</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{
border-image: url(:/Resources/escritorio2.png);
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QStackedWidget" name="menu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>399</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="pprincipal">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>27</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>27</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="botonJugar">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>27</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Jugar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonCargar">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>27</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Cargar Partida</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonSettings">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>27</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Opciones</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonSalir">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>27</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Salir </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="cargar">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="botonPartida1">
            <property name="maximumSize">
             <size>
              <width>282</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Partida guardada 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonPartida2">
            <property name="maximumSize">
             <size>
              <width>282</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Partida guardada 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonPartida3">
            <property name="maximumSize">
             <size>
              <width>282</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Partida guardada 3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonVolver3">
            <property name="maximumSize">
             <size>
              <width>282</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Volver</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="dificultad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>30</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="botonFacil">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>27</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Fácil</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonNormal">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>27</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1534;</string>
            </property>
            <property name="text">
             <string>Normal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonDemonio">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>24</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #330000;</string>
            </property>
            <property name="text">
             <string>Modo Demonio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonVolver1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>176</horstretch>
              <verstretch>20</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>271</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>25</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Volver</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="descrdificultad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>24</number>
          </property>
          <property name="rightMargin">
           <number>24</number>
          </property>
          <property name="bottomMargin">
           <number>40</number>
          </property>
          <item>
           <widget class="QLabel" name="descripcion">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>500</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonStart">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>27</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532;</string>
            </property>
            <property name="text">
             <string>Comenzar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonVolver2">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Sans Serif</family>
              <pointsize>25</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: white;
background-color: #1e1532</string>
            </property>
            <property name="text">
             <string>Volver</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
